minvws/nl-kat-coordination
# Fix boefje container image url

### Changes
The URL of the container image on the Boefje Detail page (see screenshot) referred to an incorrect URL. The text of the URL looked fine, but clicking on the link redirected to a local page (which gave a 404 error message).

### Demo

![afbeelding](https://github.com/user-attachments/assets/6ce27024-a695-4f6b-9369-db4bf4cd279c)



### QA notes
- Go to a Boefje Detail page that contains a container image (for example Nmap TCP, Nmap UDP, DNSSEC or your own boefje)
- See the changes in the Container Image section

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
